openwisp_controller/config/migrations/0033_configurable_verbose_device_name.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.1.2 on 2020-10-03 19:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('config', '0032_update_legacy_vpn_backend'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='device', options={'verbose_name': 'Device'},
```

Comment:
Here we can import `app_settings` and change this line to 
```
name='device', options={'verbose_name': app_settings.DEVICE_VERBOSE_NAME},
```

This will not create any new migrations when `OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME` is configured by user.

Reply:
I solved this problem using different approach I will add that commit soon.
In my method there is no need of migration file.